tests/python/gpu/test_fusion.py

```diff
@@ -241,13 +241,31 @@ def check_leakyrelu_ops():
     print("Checking fusion of LeakyReLU:gelu")
     check_fused_symbol(mx.sym.LeakyReLU(a+b, act_type='gelu'), a=arr1, b=arr2)
 
+def check_broadcast_ops():
+    a = mx.sym.Variable('a')
+    b = mx.sym.Variable('b')
+    arr1 = mx.random.uniform(shape=(8,8,2,3))
+    arr2 = mx.random.uniform(shape=(1,1,1,3))
```

Comment:
Please test it with other dimensions as well (like (1,8,1,3) or (8,1,2,1))

Reply:
Do you mean for arr1?

Reply:
no, arr2

Reply:
these tests were added